Urbansim Visualizer

bobbylu opened the pull request:
1. Changes have been made by me and Conor so that urbansim results would be written to MTC s3 bucket, which would be fetched for front-end visualization at urbanforecast.com
2. Currently, the slack is also setup that's working right now. But may need to come back and check for further refinement. Ideally, we can add a parser that can turn on and off the map and slack function.

theocharides commented:
@bobbylu any reason this turns off the `config` step?

bobbylu commented:
@theocharides Now it is turned on. 
I wasn't sure why it was turned off in the first place. I didn't change that part. But now it is turned on. It should work now.

theocharides commented:
@bobbylu could you explain why this setting goes under `options.scenario`?
https://github.com/BayAreaMetro/bayarea_urbansim/pull/191/commits/7a4e55f479a0b3bf2980ad9e28761079f5625b00#diff-6145e032b68cb361cc31d6274aa20c42R92

theocharides commented:
@bobbylu I updated my comment above so please check!

theocharides commented:
Hm what do you mean by "runs a futures 2 scenario".

bobbylu commented:
It starts to run s11 if I remember correctly.

Shijia Bobby Lu
215.460.0735
________________________________
From: Elizabeth Theocharides <notifications@github.com>
Sent: Wednesday, July 8, 2020 6:48:02 PM
To: BayAreaMetro/bayarea_urbansim <bayarea_urbansim@noreply.github.com>
Cc: Bobby Lu <bobbylu.shijia@gmail.com>; Mention <mention@noreply.github.com>
Subject: Re: [BayAreaMetro/bayarea_urbansim] Urbansim Visualizer (#191)


Hm what do you mean by "runs a futures 2 scenario".

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/BayAreaMetro/bayarea_urbansim/pull/191#issuecomment-655847236>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ABH2NJ567G6SPVQVGH7ZFVDR2UOVFANCNFSM4OJSBLHQ>.

theocharides commented:
You have to pass it a scenario to select that scenario. So not too sure what's going on.

bobbylu commented:
Let me try adding a new parser again tomorrow. And let you know what happens

Shijia Bobby Lu
215.460.0735
________________________________
From: Elizabeth Theocharides <notifications@github.com>
Sent: Wednesday, July 8, 2020 6:57:39 PM
To: BayAreaMetro/bayarea_urbansim <bayarea_urbansim@noreply.github.com>
Cc: Bobby Lu <bobbylu.shijia@gmail.com>; Mention <mention@noreply.github.com>
Subject: Re: [BayAreaMetro/bayarea_urbansim] Urbansim Visualizer (#191)


You have to pass it a scenario to select that scenario. So not too sure what's going on.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/BayAreaMetro/bayarea_urbansim/pull/191#issuecomment-655849689>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ABH2NJ72DVGVIU2V2T24LE3R2UPZHANCNFSM4OJSBLHQ>.

theocharides commented:
Okay, please also see my comment about `MAP`.

bobbylu commented:
@theocharides 
to close this:
(1) I removed the SLACK=MAP=True under option.scenario. I realized that there is one line of code that assign value to Slack and Map. https://github.com/BayAreaMetro/bayarea_urbansim/blob/47cb1c52fef2265d7c2e5162d33a6f4fd0abbcd7/baus.py#L25
So the correct way to do it is to add two variables to PATH:
a. URBANSIM_SLACK = True
b. SLACK_TOKEN = xxx 
One we have added this to the windows system, SLACK = MAPS = True is automatically assumed, and there is no need to call them out and assign value again. 
See more in this asana task https://app.asana.com/0/1140564422342577/1188536427565754/f

(2) RE: comment on MAP/MAPS. You are right, they are should be MAPS. Again, since I removed that line of code, it shouldn't be a problem now.

Let me know if you have any questions.

bobbylu commented:
@theocharides I also revised README.md with language on setting up AWS and slack

bobbylu commented:
@theocharides  I successfully ran the model to 2050 with these changes.

theocharides commented:
Started this work in October 2019, and we get to merge today! YAY. @bobbylu @conorhenley
